model/src/main/kotlin/LicenseFinding.kt

```diff
@@ -53,18 +53,23 @@ fun LicenseFindingsMap.removeGarbage(copyrightGarbage: CopyrightGarbage) =
  */
 data class LicenseFinding @JsonCreator constructor(
         val license: String,
+        val locations: SortedSet<TextLocation> = sortedSetOf(),
```

Comment:
I'm wonder why we should assign a default value to this, but not to `copyrights`. Sure, not all scanners might provide `locations`, but the same is true for `copyrights`. Would it require any additional code changes currently to simply omit the default value here? Or is it simply for backward compatibility for deserializing older results?

Reply:
It's for backward compatibility. It's the easiest way to ensure old results can be deserialized.

Reply:
Yes, I'd have a slight preference for that for clarity in this case.

Reply:
Done.

Reply:
I had to replace the `@JsonCreator` constructors with a custom deserializer, see the commit message for details.